core/src/com/unciv/ui/victoryscreen/VictoryScreen.kt

```diff
@@ -175,22 +177,89 @@ class VictoryScreen(val worldScreen: WorldScreen) : PickerScreen() {
 
     private fun setCivRankingsTable() {
         val majorCivs = gameInfo.civilizations.filter { it.isMajorCiv() }
-        val civRankingsTable = Table().apply { defaults().pad(5f) }
+        val playerCiv = gameInfo.civilizations.filter { it.isCurrentPlayer() }
+
+        contentsTable.clear()
+
+        if (gameInfo.gameParameters.demographicsEnabled) contentsTable.add(buildDemographicsTable(majorCivs, playerCiv))
+        else contentsTable.add(buildRankingsTable(majorCivs))
+    }
+
+    private fun buildDemographicsTable(majorCivs: List<CivilizationInfo>, playerCiv: List<CivilizationInfo>): Table {
+        val rankLabels = arrayOf("Demographic", "Rank", "Value", "Best", "Average", "Worst")
+        val demographicsTable = Table().apply { defaults().pad(5f) }
+        for (n in rankLabels.indices)   {
```

Comment:
WAYYY too much indenting in this function, which means it's hard to understand.
Why are you iterating by index instead of by value? The index is meaningless, it's the value that's important!

Separate n==0 out to a separate `for` if you think it's different enough for an if/else

Reply:
> make the changes you suggested before this gets merged

Well here it says "yes"

Reply:
Moved code for static header row to new function, simplified other code and removed several indentations.